packages/react/src/components/Notice/index.tsx

```diff
@@ -14,27 +14,38 @@ export interface NoticeProps
   type?: keyof typeof iconTypeMap;
   title: ContentNode;
   icon?: IconType;
+  variant?: 'default' | 'condensed';
   children?: ReactNode;
 }
 
 const Notice = forwardRef<HTMLDivElement, NoticeProps>(
   (
-    { type = 'info', title, icon, children, ...otherProps }: NoticeProps,
+    {
+      type = 'info',
+      title,
+      icon,
+      variant = 'default',
+      children,
+      ...otherProps
+    }: NoticeProps,
     ref
   ) => {
     return (
       <div
         className={classNames('Notice', {
-          [`Notice--${type}`]: type
+          [`Notice--${type}`]: type,
+          [`Notice--${variant}`]: variant
```

Comment:
This should be statically typed for the variant so we only get classes for non-default variants.